app/components/modal.js

```diff
@@ -0,0 +1,114 @@
+
+  return (
+    <div className="fixed inset-0 flex items-center justify-center z-50">
+      <div className="bg-gray-500 bg-opacity-75 fixed inset-0"></div>
+      <div className="bg-white relative rounded-lg max-w-2xl w-full mx-4">
+        <div className="p-6">
+          <img
+            src={event.image}
+            height={300}
+            width={300}
+            className=" ml-40 mb-6"
+          />
+          <h2 className="font-semibold leading-6 text-gray-900 text-2xl">
+            {event.title}
+          </h2>
+          <dl className="mt-4 space-y-4">
+            <div className="flex items-center">
+              <UserCircleIcon
+                className="h-6 w-5 text-gray-400"
+                aria-hidden="true"
+              />
+              <dt className="ml-2 text-sm font-medium leading-6 text-gray-900">
+                Location:
+              </dt>
+              <dd className="ml-1 text-sm leading-6 text-gray-700">
+                {event.location}
+              </dd>
+            </div>
+            <div className="flex items-center">
+              <CalendarDaysIcon
+                className="h-6 w-5 text-gray-400"
+                aria-hidden="true"
+              />
+              <dt className="ml-2 text-sm font-medium leading-6 text-gray-900">
+                Date:
+              </dt>
+              <dd className="ml-1 text-sm leading-6 text-gray-700">
+                {event.date}
+              </dd>
+            </div>
+            <div className="flex items-center">
+              <dt className="flex-none">
+                <span className="sr-only">Price</span>
+                <CreditCardIcon
+                  className="h-6 w-5 text-gray-400"
+                  aria-hidden="true"
+                />
+              </dt>
+              <dt className="ml-2 text-sm font-medium leading-6 text-gray-900">
+                Price:
+              </dt>
+              <dd className="ml-1 text-sm leading-6 text-gray-700">
+                {event.price}
+              </dd>
+            </div>
+            <div className="flex items-center">
+              <dt className="flex-none">
+                <span className="sr-only">Tickets</span>
+                <TicketIcon
+                  className="h-6 w-5 text-gray-400"
+                  aria-hidden="true"
+                />
+              </dt>
+              <dt className="ml-2 text-sm font-medium leading-6 text-gray-900">
+                Add tickets:
+              </dt>
+              <dd className="ml-1 text-sm leading-6 text-gray-700">
+                <NumericInput />
+              </dd>
+            </div>
+          </dl>
+        </div>
+        <div className="bg-gray-50 px-4 py-3 sm:flex sm:flex-row-reverse sm:px-6">
+          <button
+            type="button"
+            className="inline-flex w-full justify-center rounded-md bg-red-600 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-red-500 sm:ml-3 sm:w-auto"
```

Comment:
Las acciones positivas, se usa color verde. Para las destructivas o peligrosas, o estilo cancelar, rojo.

Siempre utiliza un color bajo (~300) `bg-...-300` para el color normal y un color mas alto (~500/600) `bg-...-600` para cuando se hace el hover. 
```suggestion
            className="inline-flex w-full justify-center rounded-md bg-green-300 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-green-500 sm:ml-3 sm:w-auto"
```

Reply:
Esto es para crear un efecto mas atractivo cuando pasas el mouse por arriba.